lib/get-exports.js

```diff
@@ -3,12 +3,26 @@
 const getEsmExports = require('./get-esm-exports.js')
 const { parse: getCjsExports } = require('cjs-module-lexer')
 const fs = require('fs')
-const { fileURLToPath } = require('url')
+const { fileURLToPath, pathToFileURL } = require('url')
 
 function addDefault(arr) {
   return Array.from(new Set(['default', ...arr]))
 }
 
+async function getFullCjsExports(url, context, parentLoad, source) {
+  const ex = getCjsExports(source)
+  return Array.from(new Set([
+    ...addDefault(ex.exports),
+    ...(await Promise.all(ex.reexports.map(re => getExports(
+	    re.startsWith('./') || re.startsWith('../')
```

Comment:
Possibly this should consider Windows `\\` dir separators, if on Windows? I'm not sure how important that is, and it could be done later.

Reply:
To be honest the entire path handling here felt hacky when I wrote it, and I'm not sure if this is the best way to do it, though I didn't immediately come up with a better option. :)

As far as windows dir separators - I suppose? I think I initially thought cjs-module-lexer would normalize this, but in retrospect that's probably not correct. This also made me realize that importing `.` or `..` would currently not be handled correctly either - I'll fix that up